docs/DeveloperGuide.md

```diff
@@ -349,7 +349,7 @@ application's requirements.
 | Emulate 2-input logic circuit | Being a binary tree, each node can have atmost 2 children, thus recreating a 2-Input Logic Gate |
 
 Therefore, since the Binary Heap-Like data structure best-fit the requirements for the system, the data structure was selected to
```

Comment:
Consider changing 'best-fit' to 'best fits'

Reply:
aight